specification/columns/negotiatedlistcost.md

```diff
@@ -0,0 +1,37 @@
+# Negotiated List Cost
```

Comment:
Maybe the name Negotiated List Cost is a bit long, and it would imply to rename the column List Cost into Public List Cost, to avoid ambiguities.
Is there any issue with the shorter name Negotiated Cost?

Reply:
+1 
Let's break this down... 

re: "Negotiated":
The vast majority of organizations don't have negotiated discounts. And given this is the basis for everything they will look at, that means all documentation will revolve around this as the way to quantify commitment savings. Given we've already differentiate between committed and non-committed (aka "on-demand") usage in PricingCategory as "On-Demand", we should stick with consistent terminology here and use "OnDemand".

re: "List":
This feels unnecessary. Everything is discounted from list, so if we add it here, wouldn't it make sense in every price/cost column? I would argue that's overkill.

Reply:
Could you please take a look at the [FOCUS 1.0 columns and negotiation activities - Definition and Naming options GSpreadsheet](https://docs.google.com/spreadsheets/d/1VlDx7wGH1atBTksQV-sR2ubxbDQjOBdt7cJsQH0SapQ/edit#gid=749340528)? It contains some options, pros, and cons. 

Any feedback, suggestions, or additional criteria would be greatly appreciated. Thank you!

Reply:
To help for the naming discussion, do we foresee other types of costs than List, Negotiated, Billed, Effective?

Reply:
Discussion was done about the name and the vote result is Contracted Cost => resolving conversion